### PR TITLE
06css

### PR DESCRIPTION
**Pull request by andrei2701**

### CODE REVIEW

File: AndreiFamishkin/01.html/task4.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+        <title>Выравнивание по центру</title>
+        <style>
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task4.html:7]: Just one tag style with all classes

File: AndreiFamishkin/02.table/My_happy_birthday.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		 table {
+        margin: auto; 
+		margin-top: 180px
+   }
+		</style>
+		</head>
+	<body>
+		<table width="200" border="1">
+			<caption>Месяц моего рождения</caption>
+			<col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67">
```
Comment (pluhin) [inline AndreiFamishkin/02.table/My_happy_birthday.html]: not comma, dot

File: AndreiFamishkin/04.MultiMedia/piton2.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+            <img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton2.html:16]: Why blank here? "blank" used to apply for external links

File: AndreiFamishkin/04.MultiMedia/piton1.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+			<img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 1</h1>
+			<ul>
+				<li><a href="./piton1.html" target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton1.html]: Why blank here? "blank" used to apply for external links

File: AndreiFamishkin/03.links/piton2.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton2.html:15]: Why blank here? "blank" used to apply for external links

File: AndreiFamishkin/03.links/piton3.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton3.html:15]: Why blank here? "blank" used to apply for external links

File: AndreiFamishkin/04.MultiMedia/piton3.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+            <img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton3.html:16]: Why blank here? "blank" used to apply for external links

File: AndreiFamishkin/05.Form/Form.html [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+												<option value="2">Минск</option>
+												<option value="3">Витебск</option>
+											</optgroup>
+											<optgroup label="Города России">
+													<option value="4">Москва</option>
+													<option value="5">Рязань</option>
+													<option value="6">Магадан</option>
+											</optgroup>
+										</select>
+										<select name="list1"  size="6" multiple id="list1">
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: If we use multiple, then need [] in name of variable

File: AndreiFamishkin/05.Form/Form.html [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: It should be label, there are no paragraphs in this task

File: AndreiFamishkin/05.Form/Form.html [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+							</form>
+					</th>
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: It should be label, there are no paragraphs in this task

File: AndreiFamishkin/05.Form/Form.html [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
+<pre><i>Телефонный номер в формате
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: where label?
It should be label, there are no pre tags in this task

File: AndreiFamishkin/05.Form/Form.html [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
++ХХХ-ХХ-ХХХХХХХ, где Х-любое число </i></pre>
+									<input type="text" name="number" pattern="\+375-(29|44|25|33)-[0-9]{7}" placeholder="+375-29-2544545">
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: Wrong pattern

File: AndreiFamishkin/05.Form/Form.html [hunk 13]
```diff
@@ -0,0 +1,174 @@
...
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
```
Comment (pluhin) [inline AndreiFamishkin/05.Form/Form.html]: Where is pattern and name here?

File: AndreiFamishkin/06css/python/python_02_link.html [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+	}
+	</style>
+	<style media="print">
+		body {
+		font-family: serif;
+		font-size: 40px;
+		color: #fff;
+		background-color: #000;
+		}
+		</style>-->
```
Comment (pluhin) [inline AndreiFamishkin/06css/python/python_02_link.html]: What is -->?

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by andrei2701** [comment]
I used "blank" to open the link in another window27.01.2019, 20:12, "Siarhei Pishchyk" <notifications@github.com>:@pluhin requested changes on this pull request.



In AndreiFamishkin/01.html/task4.html:
> @@ -0,0 +1,91 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+        <title>Выравнивание по центру</title>
+        <style>

Just one tag style with all classes



In AndreiFamishkin/02.table/My_happy_birthday.html:
> +<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Календарь</title>
+		<style>
+		 table {
+        margin: auto; 
+		margin-top: 180px
+   }
+		</style>
+		</head>
+	<body>
+		<table width="200" border="1">
+			<caption>Месяц моего рождения</caption>
+			<col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67">

not comma, dot



In AndreiFamishkin/04.MultiMedia/piton2.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+            logotip
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/04.MultiMedia/piton1.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Ссылки</TITLE>
+		<style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+			logotip
+		<h1>HOW-TO: Программа на Python, Часть 1</h1>
+			<ul>
+				<li><a href="./piton1.html" target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/03.links/piton2.html:
> @@ -0,0 +1,64 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/03.links/piton3.html:
> @@ -0,0 +1,106 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/04.MultiMedia/piton3.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+            logotip
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/05.Form/Form.html:
> +							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Города РБ</legend>
+										<select name="list" id="list">	
+											<optgroup label="Города РБ">
+												<option value="1">Брест</option>
+												<option value="2">Минск</option>
+												<option value="3">Витебск</option>
+											</optgroup>
+											<optgroup label="Города России">
+													<option value="4">Москва</option>
+													<option value="5">Рязань</option>
+													<option value="6">Магадан</option>
+											</optgroup>
+										</select>
+										<select name="list1"  size="6" multiple id="list1"> 	

If we use multiple, then need [] in name of variable



In AndreiFamishkin/05.Form/Form.html:
> +														<option value="6">Магадан</option>
+										</select>
+												<input type="submit" name="otpravit" value="Ok">
+								</fieldset>
+							</form>
+					</th>
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>

It should be label, there are no paragraphs in this task



In AndreiFamishkin/05.Form/Form.html:
> +														<option value="4">Москва</option>
+														<option value="5">Рязань</option>
+														<option value="6">Магадан</option>
+										</select>
+												<input type="submit" name="otpravit" value="Ok">
+								</fieldset>
+							</form>
+					</th>
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>

It should be label, there are no paragraphs in this task



In AndreiFamishkin/05.Form/Form.html:
> +										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
+<pre><i>Телефонный номер в формате

where label?
It should be label, there are no pre tags in this task



In AndreiFamishkin/05.Form/Form.html:
> +										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
+<pre><i>Телефонный номер в формате
++ХХХ-ХХ-ХХХХХХХ, где Х-любое число </i></pre>
+									<input type="text" name="number" pattern="\+375-(29|44|25|33)-[0-9]{7}" placeholder="+375-29-2544545">

Wrong pattern



In AndreiFamishkin/05.Form/Form.html:
> +									<legend>Upload</legend>
+										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">

Where is pattern and name here?



In AndreiFamishkin/06css/python/python_02_link.html:
> +<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+	<style media="screen">
+	body {
+		font-family: sans-serif;
+		font-size: 14px;
+		background-color: #eee;
+	}
+	</style>
+	<style media="print">
+		body {
+		font-family: serif;
+		font-size: 40px;
+		color: #fff;
+		background-color: #000;
+		}
+		</style>-->

What is -->?

**Comment by andrei2701** [comment]
"If we use multiple, then need [] in name of variable"then we see no choice (Array)GET variables:Array ( )POST variables:Array ( [list] => 2 [list1] => Array ( [0] => 3 ) [otpravit] => Ok )list = 2list1 = Arrayotpravit = OkFILE variables:27.01.2019, 20:12, "Siarhei Pishchyk" <notifications@github.com>:@pluhin requested changes on this pull request.



In AndreiFamishkin/01.html/task4.html:
> @@ -0,0 +1,91 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+        <title>Выравнивание по центру</title>
+        <style>

Just one tag style with all classes



In AndreiFamishkin/02.table/My_happy_birthday.html:
> +<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Календарь</title>
+		<style>
+		 table {
+        margin: auto; 
+		margin-top: 180px
+   }
+		</style>
+		</head>
+	<body>
+		<table width="200" border="1">
+			<caption>Месяц моего рождения</caption>
+			<col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67">

not comma, dot



In AndreiFamishkin/04.MultiMedia/piton2.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+            logotip
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/04.MultiMedia/piton1.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Ссылки</TITLE>
+		<style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+			logotip
+		<h1>HOW-TO: Программа на Python, Часть 1</h1>
+			<ul>
+				<li><a href="./piton1.html" target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/03.links/piton2.html:
> @@ -0,0 +1,64 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/03.links/piton3.html:
> @@ -0,0 +1,106 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/04.MultiMedia/piton3.html:
> +<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<META charset="utf-8">
+        <TITLE>Ссылки</TITLE>
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+            logotip
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>

Why blank here? "blank" used to apply for external links



In AndreiFamishkin/05.Form/Form.html:
> +							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Города РБ</legend>
+										<select name="list" id="list">	
+											<optgroup label="Города РБ">
+												<option value="1">Брест</option>
+												<option value="2">Минск</option>
+												<option value="3">Витебск</option>
+											</optgroup>
+											<optgroup label="Города России">
+													<option value="4">Москва</option>
+													<option value="5">Рязань</option>
+													<option value="6">Магадан</option>
+											</optgroup>
+										</select>
+										<select name="list1"  size="6" multiple id="list1"> 	

If we use multiple, then need [] in name of variable



In AndreiFamishkin/05.Form/Form.html:
> +														<option value="6">Магадан</option>
+										</select>
+												<input type="submit" name="otpravit" value="Ok">
+								</fieldset>
+							</form>
+					</th>
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>

It should be label, there are no paragraphs in this task



In AndreiFamishkin/05.Form/Form.html:
> +														<option value="4">Москва</option>
+														<option value="5">Рязань</option>
+														<option value="6">Магадан</option>
+										</select>
+												<input type="submit" name="otpravit" value="Ok">
+								</fieldset>
+							</form>
+					</th>
+			</tr>
+			<tr align="left" valign="top">
+					<th>
+							<h4>Загрузка файла на сервер:</h4>
+							<form method="POST" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Upload</legend>
+										<p>Файл:</p>

It should be label, there are no paragraphs in this task



In AndreiFamishkin/05.Form/Form.html:
> +										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
+<pre><i>Телефонный номер в формате

where label?
It should be label, there are no pre tags in this task



In AndreiFamishkin/05.Form/Form.html:
> +										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">
+<pre><i>Телефонный номер в формате
++ХХХ-ХХ-ХХХХХХХ, где Х-любое число </i></pre>
+									<input type="text" name="number" pattern="\+375-(29|44|25|33)-[0-9]{7}" placeholder="+375-29-2544545">

Wrong pattern



In AndreiFamishkin/05.Form/Form.html:
> +									<legend>Upload</legend>
+										<p>Файл:</p>
+										<input type="file"name="file_Field" dir="ltr">
+										<p>Срок хранения (года):</p>
+										<input type="number" name="srok_hranenia" maxlength="2">
+										<input type="submit" name="zagruzit" value="Загрузить"><br>
+
+								</fieldset>
+							</form>
+					</th>
+					<th>
+							<h4>Валидация данный HTML5:</h4>
+							<form method="POST" action="http://sc.it-academy.by/fe/form.php">
+								<fieldset>
+									<legend>Строка поиска</legend>
+										<input type="text" placeholder="Только русские буквы">

Where is pattern and name here?



In AndreiFamishkin/06css/python/python_02_link.html:
> +<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+	<style media="screen">
+	body {
+		font-family: sans-serif;
+		font-size: 14px;
+		background-color: #eee;
+	}
+	</style>
+	<style media="print">
+		body {
+		font-family: serif;
+		font-size: 40px;
+		color: #fff;
+		background-color: #000;
+		}
+		</style>-->

What is -->?

**Review by pluhin — APPROVED** [review]

**Comment by andrei2701** [comment]
In connection with the workload at work, I will look at the webinars in the records, as I complete the tasks I will upload to GitHub27.01.2019, 22:07, "Siarhei Pishchyk" <notifications@github.com>:@pluhin approved this pull request.

**Pull request merged**